scripts/release/set-version.sh

```diff
@@ -5,13 +5,11 @@ set -euo pipefail
 script_dir=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
 
 . $script_dir/../lib.sh
```

Comment:
At first, it confused me, that the `set-version` script also updated the toolchain, but this makes total sense. Do we maybe want to rename the file to `update-stable-regions` or something similar?

Reply:
I can't come up with a better name. It should be `update-stable-regions` because it updates regions with `dev` and `unstable` as well. I'll keep it as `set-version.sh` and re-visit this in scope of https://github.com/rust-marker/marker/issues/286